src/ralph/lib/transitions/views.py

```diff
@@ -291,3 +293,11 @@ def get_context_data(self, *args, **kwargs):
             context['are_jobs_running'] = any([j.is_running for j in jobs])
             context['for_many_objects'] = True
         return context
+
+
+class KillTransitionJobView(View, metaclass=PermissionViewMetaClass):
+
+    def get(self, request, *args, **kwargs):
+        job = TransitionJob.objects.get(pk=kwargs.get('job_id'))
```

Comment:
and don't use `get('job_id')` - it's required param for this view, so put it in method params declaration